Automattic/woocommerce-payments
# Fix changelog from PR 6225

#### Changes proposed in this Pull Request

Fix changelog from https://github.com/Automattic/woocommerce-payments/pull/6225 which we won't want to show as an entry.

There is subtle difference between changelog files that will add an entry and one that won’t.

This will add an entry:
![Screenshot 2023-05-11 at 07 57 54](https://github.com/Automattic/woocommerce-payments/assets/73803630/356369a5-0067-4317-a18d-216f1e06a655)

This won't:
![Screenshot 2023-05-11 at 07 57 45](https://github.com/Automattic/woocommerce-payments/assets/73803630/d0c1dfde-94ae-4c17-ab45-e79b2026ddf2)


I need to add a new changelog file (but still one that won't add a changelog entry in `changelog.txt` and `readme.txt`) too apparently because [the GH check looks for an *added* changelog file](https://github.com/Automattic/woocommerce-payments/blob/develop/bin/check-changelog.sh#L6-L7).

Slack thread: p1683766530045969-slack-C045PC0V40G

#### Testing instructions

* Run `npm run changelog`.
* Type `fix-5884-deposits-list-tracks` as name. Notice that is the changelog name from https://github.com/Automattic/woocommerce-payments/pull/6225.
* Choose `dev`.
* When prompted `Changelog entry. May be left empty if this change is particularly insignificant`, leave empty.
* Type `covered by PR #6188` for the `You omitted the changelog entry, which is fine. But please comment as to why no entry is needed` prompt.
* Expect no file changed.

-------------------


**Post merge**
